main.py

```diff
@@ -1,63 +1,11 @@
-from fastapi import FastAPI, HTTPException
-from pydantic import BaseModel 
+from fastapi import FastAPI
 import models 
-from sqlalchemy.orm import Session
-from fastapi import Depends
-from database import engine, get_db
+from database import engine
 import uvicorn
 
-
 models.Base.metadata.create_all(bind=engine)
```

Comment:
This should be moved to a create_db() function inside database.py file.
Then, in the main.py you should just call the create_db() function with the @app.on_event("startup") decorator